### ISSUE TITLE
Should private license offers expire for inactivity?

### ISSUE THREAD

**Issue by kemitchell** [licensezero/licensezero-faq#1]
Should licensezero.com stop offering private licenses for projects that haven't been offered, or sold a private license, in some set period of time?

For example:

1. A offers project X through licensezero.com on January 1, 2020.
2. licensezero.com sells a private license for X on June 1, 2020.
3. A offers X through licensezero.com on January 1, 2021.
4. Some number of days before January 1, 2022, licensezero.com sends the developer a notification.
5. On January 1, 2022, licensezero.com automatically stops offering private licenses, and marks the project "retracted", since nobody has bought a private license in one year's time.

Background: License Zero's public licenses already revert, automatically, to permissive terms if private licenses aren't available for 90 calendar days. This approach would trigger that reversion---implemented, legally, as a waiver of the conditions limiting noncommercial or non-Open Source use---on a time limit.

The general effect of this change would probably be to reduce the number of "orphan" License Zero projects for which nobody wants a private license, but remain subject to noncommercial use or non-Open Source use.  That's good.

On the other hand, some developers might see their projects reverted to permissive terms accidentally. Some projects might also release "before their time", and become popular later, after they've reverted to permissive terms.

**Comment by cristim** [comment]
I am in favor of this, but I think there should be some guardrails to avoid accidents.

After each year of inactivity the author should be proposed to do it, and asked for approval. The author should have the right to postpone it a few times, but if the project is dormant 
for a longer period, maybe 5 years in total, the project would be released under a permissive license.

**Comment by kemitchell** [comment]
@cristim I agree on "guardrails". And that's a very good way to describe what we want.

**Comment by kemitchell** [comment]
This question is complicated a bit with the option for a reciprocal license. The reciprocal license should provide enough permission that folks are comfortable using it without any automatic waiver.
